v2/cmd/samples/http/raw/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
```

Comment:
I added this back in, it is extremely useful for debugging.

Reply:
Can we have it somewhere else? it's not really an sdk sample...

Reply:
maybe I move it to `cmd/tools`

Reply:
fine for me :+1: